web/admin/src/Admin.js

```diff
@@ -320,6 +370,11 @@ export default class Admin extends PureComponent {
     return true
   }
 
+  adminScreenUrlSave = () =>
```

Comment:
"Save" in the name makes this seem like this function is doing something, rather than just returning something without side effects.